cli/namespace.go

```diff
@@ -69,41 +69,46 @@ func parseNamespaceDataKVs(namespaceDataStr string) (map[string]string, error) {
 func newNamespaceCommands() []*cli.Command {
 	return []*cli.Command{
 		{
-			Name:  "describe",
-			Usage: "Describe a Namespace by name or Id",
-			Flags: describeNamespaceFlags,
+			Name:      "describe",
+			Usage:     "Describe a Namespace by name or Id",
+			Flags:     describeNamespaceFlags,
+			ArgsUsage: "namespace-name",
```

Comment:
```suggestion
			ArgsUsage: "[NAMESPACE NAME]",
```

for consistency with https://github.com/temporalio/tctl/blob/1ec998d5dded6b3134ee68d2b5e37c3e2d182777/cli/config.go#L41